app/models/user_role.rb

```diff
@@ -0,0 +1,6 @@
+class UserRole < ApplicationRecord
```

Comment:
Rolify by default uses `has_and_belongs_to_many` relationships. I broke away from this using has_many with through relationships instead. Thus creating the join table model itself and probably primary keying it.